Monika After Story/game/script-affection.rpy

```diff
@@ -341,13 +341,25 @@ init 15 python in mas_affection:
         """
         Runs when transitioning from affectionate to enamored
         """
+
+        # If the greeting hasn't been seen yet, push the islands greeting
+        if not store.seen_event("greeting_ourreality"):
+            store.unlockEventLabel("greeting_ourreality",eventdb=evhand.greeting_database)
+
+        # lock islands event
+        store.unlockEventLabel("mas_monika_islands")
```

Comment:
if i reach enamored, I can view the islands event without seeing the greeting first.